prow/config/config.yaml

```diff
@@ -513,13 +522,6 @@ tide:
     "*": https://prow.tidb.io/pr
 
   queries:
-    - repos:
-        - pingcap/tidb-dashboard
-        - pingcap/tidb-operator
-      labels:
-        - status/can-merge
-      missingLabels:
-        - needs-rebase
```

Comment:
Why delete this? It seems that the WIP and hold plugins have not been enabled in these two repositories.

Reply:
I think it's OK. Since there is no deployment, there will be no such labels, so that merging queries will reduce more API usage.

Reply:
OK